client/js/widgets/widget.js

```diff
@@ -1831,7 +1831,7 @@ export class Widget extends StateManaged {
           a.turnCycle = 'forward'
         }
         let c = a.source === 'all' ? Array.from(widgets.values()) : collections[getCollection(a.source)] || [];
-        c = c.filter(w => w.get('type') === 'seat');
+        c = c.filter(w => w.get('type') === 'seat' && w.get('skipTurn') !== true);
```

Comment:
I would just do `!w.get('skipTurn')`. If someone sets this with a routine which happens to have `0` or `1`, I don't see a reason why it shouldn't work.

Reply:
>  would just do !w.get('skipTurn'). If someone sets this with a routine which happens to have 0 or 1, I don't see a reason why it shouldn't work.

Good idea.  Done.